{"Launchpad":"https://bugs.launchpad.net/bugs/1381943","Id":"1381943","Gerrit":"https://review.openstack.org/130595","IdGerrit":"130595","IdCommit":"6338b76a2de7d50d4fc47d8d88f2ba60cf15d939","Files":["cinder/tests/test_solidfire.py","cinder/volume/drivers/solidfire.py","cinder/volume/manager.py"],"Lines_Inseted":14,"Lines_Deleted":0,"CommitParent":"c10525973651d488f5d7d83446561406742f61d4","Classification":"Bug","KeywordsTitle":"migrated, can't be attached","Description":"reproduce, migrate, would raise, exception, ERROR, was not found, unfortunately.","KeywordsCommit":"Fix on migrated, creating, works, deleting, missed, change adds, check on, to avoid, won't change, ","Comments":"It's a bug report.In the description we have how to reproduce the bug."}